tokio/src/process/mod.rs

```diff
@@ -330,6 +330,14 @@ impl Command {
         &mut self.std
     }
 
+    /// Cheaply convert into a `std::process::Command`.
+    ///
+    /// Note that Tokio specific options (currently only option [`kill_on_drop`](Command::kill_on_drop))
+    /// will be lost.
```

Comment:
This is a bit awkward to read. How about:
```suggestion
    /// Note that Tokio specific options will be lost. Currently, this only applies to [`kill_on_drop`].
    ///
    /// [`kill_on_drop`]: Command::kill_on_drop
```

Reply:
Yes, this is indeed a better document! Thank you for your suggestion!